### ISSUE TITLE
Improvement idea: cache emails for remote IMAP servers

### ISSUE THREAD

**Issue by ninja-** [RainLoop/rainloop-webmail#935]
Hey there, 
I am using rainloop for both my local mail server and my remote inbox.
Navigating the remote emails is really slow because it seems like it needs to make a new connection every time you browse something or load rainloop.
My suggestion is to cache these emails like mobile/desktop apps would do.

**Comment by stephan13360** [comment]
If this would be considered please make it optional. I have a very fast connection between rainloop and my remote imap server and caching would only get annoying if i move mails from different clients.

**Comment by mikerockett** [comment]
Interestingly, I've thought of this, too. I have six IMAP servers local to me, and so the connection time is quick enough - tad slow, but not enough to complain about. But connecting to outlook.com is really slow. Takes up to 15 seconds to fetch the inbox headers. Caching would be great.

**Comment by Aidenir** [comment]
Yes, i would also like this. I'm using rainloop for both my local email server, but also for my icloud mail, and its incredibly slow even though my server is on a gigabit line. If I start maving mails between folders my lighttpd always hangs and then the haproxy marks the web server as down and I get 504 Gateway Timeouts :(
